core/src/test/scala/com/banno/kafka/ConsumerAndProducerApiSpec.scala

```diff
@@ -89,8 +89,8 @@ class ConsumerAndProducerApiSpec
         )
         .unsafeRunSync()
 
-      rms.size should ===(strings.size)
-      rms.forall(_.topic == topic) should ===(true)
+      rms.size should ===(strings.size): Unit
```

Comment:
This is a really funky mix of syntax.

Reply:
I tried to convert this to MUnit before but the tests were all goofed up after that and somebody reverted it. 🤷‍♂️ This test library did encourage some weird-looking assertions.